src/main.rs

```diff
@@ -48,7 +48,7 @@ fn dump(state: &TockilatorState) -> Result<(), Box<dyn Error>> {
     Ok(())
 }
 
-fn main() -> Result<(), Box<dyn Error>> {
+fn main_core() -> Result<(), Box<dyn Error>> {
```

Comment:
The canonical name for this in the ecosystem is `try_main` -- see https://github.com/search?l=Rust&q=try_main&type=Code